app/containers/Login.js

```diff
@@ -85,12 +95,32 @@ class Login extends Component {
 
         <TouchableOpacity onPress = {this._onPressSignInButton.bind(this)}>
           <Image
-            style = {Styles.sign_in_button}
+            style = {[Styles.sign_in_button, {marginBottom: 15}]}
```

Comment:
để cái marginBottom trong style đi cho nó thống nhất

Reply:
o` ok :))
vs logo nào logo FB sign in hay logo app :))

Reply:
logo cua fb do :))